PicPi/PictureFrameDetailsThumbnailCollectionViewCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  PictureFrameDetailsThumbnailCollectionViewCell.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 1/2/21.
+//
+
+import UIKit
+
+class PictureFrameDetailsThumbnailCollectionViewCell: UICollectionViewCell {
+  
+  var imageView: UIImageView = {
+    let imageView = UIImageView()
+    imageView.contentMode = .scaleToFill
+    imageView.isUserInteractionEnabled = true
+    imageView.translatesAutoresizingMaskIntoConstraints = false
+    
+    return imageView
+  }()
+  
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    setupSubviews()
+  }
+  
+  @available(*,unavailable)
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  
+  // MARK: - Private
+  
+  private func setupSubviews() {
+    self.backgroundColor = .cyan
```

Comment:
let's try an array of colors (I don't mind wha colors lol) and using `randomElement()!` to pick one for each cell

Might need to move this background color to the `prepareForReuse` method (I'm guessing collection view cells have this like table view cells)

Reply:
Done!
you still need to set bg color in init for the cell that has not been reused yet!